### ISSUE TITLE
`invalid config: not a dict` What should I do?

### ISSUE THREAD

**Issue by anishmittal2020** [adrienverge/yamllint#201]
- My Yamllint config is blank.

```bash
cat ~/.config/yamllint/config

```

- Input File: `.gitlab-ci.yml`

```yaml
# This file is a template, and might need editing before it works on your project.
# Full project: https://gitlab.com/pages/plain-html
pages:
  stage: deploy
  script:
    - mkdir .public
    - cp -r * .public
    - mv .public public
  artifacts:
    paths:
      - public
  only:
    - master
```

```bash
$ yamllint .gitlab-ci.yml 
invalid config: not a dict
```

What should I do? What is dict?

**Comment by adrienverge** [comment]
Hi @anishmittal2020, if your config file contains an empty string, it's not valid YAML, and yamllint doesn't accept it as a configuration.

If you don't use a custom conf in `~/.config/yamllint/config`, I suggest removing the file. Or, you could just put `{}` inside it.
